docs/appkit/ios/core/installation.mdx

```diff
@@ -49,6 +49,9 @@ array of that target:
 </TabItem>
 <TabItem value="cocoapods" label="Cocoapods">
 
+> [!WARNING]  
+> Cocoapods support may be deprecated soon, use SPM instead.
+
```

Comment:
```suggestion
:::warning
Cocoapods support may be deprecated soon, use SPM instead.
:::
```

Reply:
to get something like this -> https://reown-docs-git-llbartekll-patch-3-reown-com.vercel.app/appkit/react/core/installation